concent_api/conductor/tasks.py

```diff
@@ -151,3 +156,52 @@ def upload_acknowledged(
         f'Result_file_size: {result_file_size}',
         f'Result_package_hash: {result_package_hash}'
     )
+
+
+def store_verification_request_and_blender_subtask_definition(
+    subtask_id: str,
+    source_package_path: str,
+    result_package_path: str,
+    output_format: str,
+    scene_file: str,
+):
+    verification_request = VerificationRequest(
+        subtask_id=subtask_id,
+        source_package_path=source_package_path,
+        result_package_path=result_package_path,
+    )
+    verification_request.full_clean()
+    verification_request.save()
+
+    blender_subtask_definition = BlenderSubtaskDefinition(
+        verification_request=verification_request,
+        output_format=BlenderSubtaskDefinition.OutputFormat[output_format].name,
+        scene_file=scene_file,
+    )
+    blender_subtask_definition.full_clean()
+    blender_subtask_definition.save()
+
+    return (verification_request, blender_subtask_definition)
+
+
+def store_frames(
+    blender_subtask_definition: BlenderSubtaskDefinition,
+    frame_list: List[int],
+):
+    for frame in frame_list:
+        store_frame = Frame(
+            blender_subtask_definition=blender_subtask_definition,
+            number=frame,
+        )
+        store_frame.full_clean()
+        store_frame.save()
+
+
+def filter_frames_by_blender_subtask_verification(blender_subtask_definition: BlenderSubtaskDefinition):
+    frames_query = Frame.objects.filter(
```

Comment:
Actually this can be achieved by single query, using `.fields('number')` and `.values_list()`

Reply:
Done